### ISSUE TITLE
Return value of ...getCommentContent must be of the type string, boolean returned

### ISSUE THREAD

**Issue by thinkspill** [nunomaduro/phpinsights#62]
Running against a relatively small Laravel 5.7 app:

```
$ php artisan insights
  96/126 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░]  76%
In EmptyCommentSniff.php line 153:

  Return value of SlevomatCodingStandard\Sniffs\Commenting\EmptyCommentSniff::getCommentContent() must be of the type string, boolean returned
```

**Comment by akosglue** [comment]
Same reproduced on laravel 5.8. with a small app:
 56/60 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░]  93%
   Symfony\Component\Debug\Exception\FatalThrowableError  : Return value of SlevomatCodingStandard\Sniffs\Commenting\EmptyCommentSniff::getCommentContent() must be of the type string, boolean returned

**Comment by nunomaduro** [comment]
It's possible to have the source code of your apps? Is open source? Otherwise I can't debug this.

**Comment by akosglue** [comment]
Just before the error: $commentEndPointer is -1
at that point the file is routes/api.php with this content:
`<?php

use Illuminate\Http\Request;

/*
|--------------------------------------------------------------------------
| API Routes
|--------------------------------------------------------------------------
|
| Here is where you can register API routes for your application. These
| routes are loaded by the RouteServiceProvider within a group which
| is assigned the "api" middleware group. Enjoy building your API!
|
*/

/*

Route::middleware('auth:api')->get('/user', function (Request $request) {
    return $request->user();
});

*/`


IF i run again with removed comment:
`<?php

use Illuminate\Http\Request;

/*
|--------------------------------------------------------------------------
| API Routes
|--------------------------------------------------------------------------
|
| Here is where you can register API routes for your application. These
| routes are loaded by the RouteServiceProvider within a group which
| is assigned the "api" middleware group. Enjoy building your API!
|
*/

Route::middleware('auth:api')->get('/user', function (Request $request) {
    return $request->user();
});

`

everything works fine.

I hope this helps a little :-)

**Comment by akosglue** [comment]
Well, as it is not readable:

FAIL: https://imgur.com/ykAuUbD

OK: https://imgur.com/5xnGLM9

**Comment by slovenianGooner** [comment]
Have the same issue. Looks like empty comments that don't have a description are the problem. While @akosglue can find the root of his problem, I can't.

**Comment by nunomaduro** [comment]
Source code with the issue: https://github.com/slevomat/coding-standard/blob/8f0ec5009cf9c86d207852f6f7644e34145167e4/SlevomatCodingStandard/Sniffs/Commenting/EmptyCommentSniff.php#L153

**Comment by jadamec** [comment]
Have the same issue.

**Comment by jadamec** [comment]
Same issue here. :/

**Comment by nunomaduro** [comment]
The issue happens when you have non closed docs.

**Comment by jadamec** [comment]
As @akosglue already mentioned, this problem appears when the middleware in routes/api.php is commented as shown in screenshoots. After fixing this, there's another error #59

**Comment by nunomaduro** [comment]
This issue has been fixed on the version `v1.8.1`.
